content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: blog
+title: 'Kubernetes 1.33: Fine-grained SupplementalGroups control graduates to beta'
+date: 2025-04-23
+draft: true
+slug: fine-grained-supplementalgroups-control-beta
+author: >
+  Shingo Omura (LY Corporation)
+
+---
+
+Pod supplemental groups policy control for Pods is now beta. Read on to learn how it will help your cluster security, and how to start using it.
```

Comment:
This is a bit generic, and the latter sentence can be completely dropped. Perhaps we could highlight how this is about the "fine-grained" control provided for better security, and combine it with the sentence below?

Reply:
Thanks for this suggestion. It makes sense. How about [this](tbd)??

Reply:
fixed in [f403e82](https://github.com/kubernetes/website/pull/49944/commits/f403e82b11d9e170122a4222cd4cb2be21bf3c8e)